.coveragerc

```diff
@@ -712,6 +712,8 @@ omit =
     homeassistant/components/tts/picotts.py
     homeassistant/components/vacuum/mqtt.py
     homeassistant/components/vacuum/roomba.py
+    homeassistant/components/vacuum/xiaomi_miio.py
```

Comment:
Why are you touching xiaomi_miio.py ?

Reply:
This was a rebase mistake, fixed in the next rev.